src/views/wallet/WalletViewNav.style.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { palette } from 'styled-theme'
+
+import Menu from '../../components/shared/Menu'
+
+export const StyledMenu = styled(Menu)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-bottom: 0.5px solid #969dab;
+
+  .ant-menu-item:hover,
+  .ant-menu-submenu:hover,
+  .ant-menu-item-active,
+  .ant-menu-submenu-active,
+  .ant-menu-item-open,
+  .ant-menu-submenu-open,
+  .ant-menu-item-selected,
+  .ant-menu-submenu-selected {
+    border-bottom: 2px solid ${palette('primary', 0)};
+  }
+
+  .ant-menu-item {
+    color: ${palette('text', 0)};
+    font-family: 'MainFontSemiBold';
+    font-size: 16px;
```

Comment:
Text of tab items needs to be uppercase, see https://www.figma.com/file/LgRqZCGMShZ2mJ702MIqyV/ASGARDEX-2020?node-id=1306%3A7469